stdlib/LinearAlgebra/src/blas.jl

```diff
@@ -113,10 +113,8 @@ Set the number of threads the BLAS library should use.
 """
 function set_num_threads(n::Integer)
```

Comment:
I just noticed that on exotic BLAS this fails silently. Should  we throw an error instead?